### DISCUSSION TITLE
SKR Mini E3 V3.0 STM32G0B1

### DISCUSSION THREAD

**Discussion by FlyBoy33** [mriscoc/Ender3V2S1#905] [Q&A]
I noticed in the requirements, it specified SKR Mini E3 V3 for Ender series printers with a STM32F103 or STM32F401 SoC.
Reading my processor on my SKR it says STM32G0B1
I am brand new to this depth of firmware upgrades, so please help. 
I just installed this on my E3V2 Neo and it is amazing!!! Now I have two new BTT SKR Mini E3 V3's and two new BTT Screens {TFT35 ENC V1.0}
Can I go through the same process of downloading this firmware onto these boards AND use the screens mentioned above?
If so, what files do I need to select, and and additional instructions would be GREATLY appreciated!! Never posted in any tech Q&A before..
![3](https://user-images.githubusercontent.com/132791627/236966062-ae10d1a0-ec3f-4e09-941f-bca96a1e5c49.jpg)
![1](https://user-images.githubusercontent.com/132791627/236966064-342ec6c1-4011-4a67-a2d9-58fff4fc78b8.jpg)
![2](https://user-images.githubusercontent.com/132791627/236966065-2a9c8cee-a69e-482a-a184-30d0c25f4f7c.jpg)

**Comment by mfraser** [comment]
As I've got one of those boards arriving soon, this is going to be 9f interest to me.

I found this page comparing the various STM32 chips https://en.m.wikipedia.org/wiki/STM32

**Reply by mfraser** [reply]
Looks like it is already configured for this board, if you look at  https://github.com/mriscoc/Special_Configurations/blob/main/Ender3V2-SKRME3V3-BLTUBL-MPC/platformio.ini it says:
default_envs = STM32G0B1RE_btt

**Reply by FlyBoy33** [reply]
Oh man!! I hope this is the case!! Thank you so much for looking into it. If you come up with any further info, please comment! Thanks again!

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/discussions/131

**Reply by FlyBoy33** [reply]
Sorry...what does this mean?
